setup.py

```diff
@@ -14,8 +14,8 @@ def read(fname):
     description = ("A python program to count the number of instances of different specifications."),
```

Comment:
This file should be removed; all the information here should be moved to `pyproject.toml` if it isn't already there (the format may change slightly, check the Poetry `pyproject.toml` [documentation](https://python-poetry.org/docs/pyproject/).

Reply:
@solisa986